File: src/main/java/kitchenpos/menus/tobe/infra/JpaMenuGroupRepository.java
package kitchenpos.menus.tobe.infra;

import java.util.UUID;

import kitchenpos.menus.tobe.domain.menugroup.MenuGroup;
import kitchenpos.menus.tobe.domain.menugroup.MenuGroupRepository;
import org.springframework.data.jpa.repository.JpaRepository;

public interface JpaMenuGroupRepository
    extends MenuGroupRepository, JpaRepository<MenuGroup, UUID> {}

File: src/main/java/kitchenpos/menus/tobe/infra/JpaMenuGroupRepository.kt
package kitchenpos.menus.tobe.infra

import kitchenpos.menus.tobe.domain.MenuGroup
import kitchenpos.menus.tobe.domain.MenuGroupRepository
import org.springframework.data.jpa.repository.JpaRepository
import java.util.UUID

interface JpaMenuGroupRepository : MenuGroupRepository, JpaRepository<MenuGroup, UUID>
